lib/tools/searchWeb/getSearchWebTool.ts

```diff
@@ -1,33 +1,65 @@
 import { z } from "zod";
 import { tool } from "ai";
-import performChatCompletion from "@/lib/perplexity/performChatCompletion";
+import { searchPerplexity } from "@/lib/perplexity/searchApi";
+import { formatSearchResultsAsMarkdown } from "@/lib/perplexity/formatSearchResultsAsMarkdown";
+import type { SearchProgress } from "./types";
 
-const getSearchWebTool = (model: string = "sonar-pro") => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const getSearchWebTool = (_model?: string) => {
```

Comment:
YAGNI principle
- actual: includes unused variable with linter ignore comment
- required: remove the unused variable. this is a valid typescript linter warning.

```suggestion
const getSearchWebTool = () => {
```